pkg/schemacmp/table_test.go

```diff
@@ -513,3 +516,20 @@ func (s *tableSchema) TestJoinSchemas(c *C) {
 		}
 	}
 }
+
+func (s *tableSchema) TestTableString(c *C) {
+	ti, err := s.toTableInfo("CREATE TABLE tb (a INT, b INT)")
+	c.Assert(err, IsNil)
+
+	charsets := []string{"", mysql.DefaultCharset}
+	collates := []string{"", mysql.DefaultCollationName}
+	for _, charset := range charsets {
+		for _, collate := range collates {
+			ti.Charset = charset
+			ti.Collate = collate
+			sql := strings.ToLower(schemacmp.Encode(ti).String())
```

Comment:
we might test this SQL could be successfully parsed

Reply:
done